fnl/mods/tools/telescope.fnl

```diff
@@ -191,7 +191,7 @@
 
 (var magic
   (fn [_opts]
-    (var opts (or _opts {}))
+    (var opts (deep-copy (or _opts {})))
```

Comment:
This was causing `_opts` passed as input, which was `ivy_config` to be modified (lua uses references for all vars). This was adding additional fields breaking the preview

<img width="766" alt="Screenshot 2023-03-31 at 01 07 42" src="https://user-images.githubusercontent.com/19249682/228984129-d7336c31-6ee0-40ba-b60f-10fd4cba635d.png">